src/Components/ProductItem/ProductItem.scss

```
@@ -1,231 +1,78 @@
-            padding-right: 20px;
-            text-align: end;
-
-            &:hover {
-              color: #ff9e36;
-            }
-          }
-        }
-      }
+    .sale {
+      right: 5px;
+      background-color: #5da947;
+      border-radius: 2rem;
     }
-
-    .productLists {
-      display: flex;
-      max-width: 1197px;
-      margin: 0 auto;
-      flex-wrap: wrap;
-      column-count: 4;
-
-      .productItem {
-        padding: 0 7px;
-        margin-bottom: 25px;
-
-        .thum {
-          position: relative;
-          display: block;
-          margin-bottom: 16px;
-          background-color: $category-background-color;
-
-          .productImg {
-            border-radius: 10px;
-            width: 282px;
-            height: 282px;
-          }
-
-          .label {
-            position: absolute;
-            top: 5px;
-            padding: 5px 13px;
-            color: #fff;
-            text-align: center;
-          }
-
-          .sale {
-            right: 5px;
-            background-color: #5da947;
-            border-radius: 2rem;
-          }
-
-          .soldOut {
-            right: 5px;
-            width: 45px;
-            height: 45px;
-            padding: 11px 1px;
-            background-color: rgb(190, 190, 190);
-            border-radius: 100px;
-            font-size: 12px;
-          }
-
-          .set {
-            right: 5px;
-            padding: 8px 15px;
-            background-color: #f9ca1e;
-            border-radius: 2rem;
-          }
-
-          .new {
-            left: 5px;
-            width: 45px;
-            height: 45px;
-            padding: 15px 3px;
-            background-color: #88aaab;
-            opacity: 0.8;
-            border-radius: 100px;
-            font-size: 12px;
-          }
-        }
-
-        .itemDesc {
-          position: relative;
-
-          .itemTitle {
-            font-size: 18px;
-            margin-bottom: 16px;
-          }
-          .fa-heart {
-            position: absolute;
-            right: 0;
-            top: 0;
-            font-size: 1.4rem;
-            color: #e1e1e1;
-            cursor: pointer;
-          }
-          .yellow {
-            color: #ffdf00;
-          }
-          .itemPrice {
-            font-size: 20px;
-          }
-        }
-      }
+    .soldOut {
+      right: 5px;
+      width: 45px;
+      height: 45px;
+      padding: 11px 1px;
+      background-color: rgb(190, 190, 190);
+      border-radius: 100px;
+      font-size: 12px;
+    }
+    .set {
+      right: 5px;
+      padding: 8px 15px;
+      background-color: #f9ca1e;
+      border-radius: 2rem;
+    }
+    .new {
+      left: 5px;
+      width: 45px;
+      height: 45px;
+      padding: 15px 3px;
+      background-color: #88aaab;
+      opacity: 0.8;
+      border-radius: 100px;
+      font-size: 12px;
     }
   }
-  .hide {
-    display: none;
+  .itemDesc {
+    position: relative;
+    .itemTitle {
+      font-size: 18px;
```

```suggestion
  .itemDesc {
    position: relative;
    
    .itemTitle {
      font-size: 18px;
```
새로운 셀렉터 앞에서는 부디 한줄 띄워주세요.. ㅠ

---

셀렉터 앞에 한줄 띄워줬습니다 ㅠㅠ 다른 팀원 파일이라 못 봤네요!